src/v/coproc/tests/utils/wasm_event_generator.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020 Vectorized, Inc.
+ *
+ * Licensed as a Redpanda Enterprise file under the Redpanda Community
+ * License (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * https://github.com/vectorizedio/redpanda/blob/master/licenses/rcl.md
+ */
+
+#pragma once
+#include "bytes/bytes.h"
+#include "coproc/wasm_event.h"
+#include "model/record.h"
+#include "model/record_batch_reader.h"
+
+#include <seastar/core/sstring.hh>
+
+#include <optional>
+
+/// \brief Pass this to a generator to recieve a genned event type of your
+/// choice
+enum class make_event_type { deploy, remove, random };
```

Comment:
`make_*` is typically only used for generator functions.  .... also why not put all of these inside the `coproc` namespace?